docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+  control how much data is replicated and where that replication traffic takes place.
+- Organizational Unit (OU)
+    - OU’s provide a way to group resources so that they can be managed as a single unit. This allows admins to apply
+group policies across many computers or control access of resources for many users. OUs can also be delegated to 
+specific admins, so that they only have control over their subset of resources.
+
+Other core components of an Active Directory are
+
+- DNS Support
+    - In Active Directory, DNS is the means by which directory clients locate, or discover, domain controllers. 
+      The primary components of the architecture for DNS support of Active Directory include the domain controller
+      Locator, Active Directory domain names in DNS, and Active Directory DNS objects.
+    - Every Active Directory domain has a DNS domain name (for example, [cohovineyard.com](http://cohovineyard.com/)),
+      and every domain joined computer has a DNS name (for example, 
+      [server1.cohovineyard.com](http://server1.cohovineyard.com/)). Architecturally, domains and computers are
+      represented both as objects in Active Directory and as nodes in DNS.
+- Active Directory Schema
+    - The schema contains definitions of objects which can store information in the Active Directory. It provides the 
+      attributes associated with particular resources.
+    - Different Windows versions will have different schema versions, when different versions of windows are used to 
+      comprise the Active Directory server they will agree upon a shared Schema.
+- The Data store
+    - A data store comprises three components. The first being a collection of interfaces used to communicate with the
+      store, the second being the underlying services and logic which perform CRUD on the store, and the third is the
+      literal data. Each data store is represented as a single file replicated across all domain controllers.
+    - Proper logical structuring of an Active Directory will ensure that Domain Controllers will only have access to the
+      data required for their domain, keeping replication quick and simple. Improper structuring can saturate the
+      network with replication data. Properly structuring an on premise Active Directory is an art.
+
+### The Components and Interfaces of Active Directory Storage
+
+There are several storage components that come together to form the data handling of Active Directory
+
+The core interfaces used to communicate with the storage layer are
+
+- LDAP, Lightweight Directory Access Protocol
+- Replication within the domain controller management interface
+- A Messaging API
+- A Security Accounts Manager (SAM)
+
+The core components of the storage layer are
+
+- Directory System Agent (DSA)
+- The Database, typically represented as a single file replicated across all Domain Controllers
+- Extensible Storage Engine (ESE)
+
+![Data Store Architecture](../media/Untitled%201.png)
```

Comment:
Again, I think this needs an analogy or some more specific explanation to make this more accessible.